<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>mingw-OpenEXR.git - mingw-OpenEXR</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/mingw-OpenEXR.git/atom/openexr-2.1.0_aligned-malloc.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='mingw-OpenEXR.git' href='/cgit/mingw-OpenEXR.git/'>mingw-OpenEXR.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>mingw-OpenEXR</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/mingw-OpenEXR.git/'>summary</a><a href='/cgit/mingw-OpenEXR.git/refs/'>refs</a><a href='/cgit/mingw-OpenEXR.git/log/openexr-2.1.0_aligned-malloc.patch'>log</a><a class='active' href='/cgit/mingw-OpenEXR.git/tree/openexr-2.1.0_aligned-malloc.patch'>tree</a><a href='/cgit/mingw-OpenEXR.git/commit/openexr-2.1.0_aligned-malloc.patch'>commit</a><a href='/cgit/mingw-OpenEXR.git/diff/openexr-2.1.0_aligned-malloc.patch'>diff</a><a href='/cgit/mingw-OpenEXR.git/stats/openexr-2.1.0_aligned-malloc.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/mingw-OpenEXR.git/log/openexr-2.1.0_aligned-malloc.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/mingw-OpenEXR.git/tree/'>root</a>/<a href='/cgit/mingw-OpenEXR.git/tree/openexr-2.1.0_aligned-malloc.patch'>openexr-2.1.0_aligned-malloc.patch</a></div><div class='content'>blob: 6dd66f6d5ad778861b84ceb0d1aaaf8e94901715 (<a href='/cgit/mingw-OpenEXR.git/plain/openexr-2.1.0_aligned-malloc.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
</pre></td>
<td class='lines'><pre><code>diff -rupN openexr-2.1.0/IlmImf/ImfSystemSpecific.h openexr-2.1.0-new/IlmImf/ImfSystemSpecific.h
--- openexr-2.1.0/IlmImf/ImfSystemSpecific.h	2013-10-21 21:02:22.000000000 +0200
+++ openexr-2.1.0-new/IlmImf/ImfSystemSpecific.h	2013-12-27 01:57:30.936052388 +0100
@@ -60,8 +60,7 @@ static bool GLOBAL_SYSTEM_LITTLE_ENDIAN
 
 static void* EXRAllocAligned(size_t size, size_t alignment)
 {
-    void* ptr = 0;
-    posix_memalign(&amp;ptr, alignment, size);
+    void* ptr = _aligned_malloc(alignment, size);
     return ptr;
 }
 
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-08-28 07:23:42 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
